sdk/src/main/java/io/dapr/client/domain/CloudEvent.java

```diff
@@ -127,7 +161,89 @@ public CloudEvent(
     this.datacontenttype = "application/octet-stream";
     this.binaryData = binaryData == null ? null : Arrays.copyOf(binaryData, binaryData.length);;
   }
-    
+
+  /**
+   * Instantiates a CloudEvent.
+   * @param id              Identifier of the message being processed.
+   * @param source          Source for this event.
+   * @param type            Type of event.
+   * @param specversion     Version of the event spec.
+   * @param datacontenttype Type of the payload.
+   * @param data            Payload.
+   * @param pubsubname      Pubsub component name.
+   * @param topic           Topic name.
+   * @param time            Time.
+   * @param traceid         Trace id.
+   * @param traceparent     Trace parent.
+   * @param tracestate      Trace state.
+   */
+  public CloudEvent(

```

Comment:
This is a breaking change. Please, remove this constructor and leave the default. A constructor this long is not that readable either.

Reply:
Sure, I will remove the 2 new constructors I added. The old default ones still exist so that shouldn't have broken anything.